### PR TITLE
applications: asset_tracker_v2: Remove nRF cloud disclaimer from docs

### PR DESCRIPTION
**Pull request by simensrostad**
Remove nRF cloud disclaimer from docs. This issue has been resolved.

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]

File: applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst [hunk 1]
```diff
@@ -368,12 +368,6 @@ Support for nRF Cloud
 Enabling full support for nRF Cloud is currently a work in progress.
 Following are the current limitations in the nRF Cloud implementation of the Asset Tracker v2:
```
Comment (umapraseeda) [inline applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst:369]: There is only one limitation. May be we should rephrase this as:
"The nRF Cloud implementation of the Asset Tracker v2 application has currently the limitation that the nRF Cloud web application does not support the manipulation of real-time configurations.
However, this is possible by using the REST API calls described in `nRF Cloud Patch Device State`_.
  To manipulate the device configuration, the ``desired`` section of the device state must be populated with the desired configuration of the device."

**Review by umapraseeda — APPROVED** [review]
